src/kubernetes.cc

```diff
@@ -1042,27 +1042,43 @@ json::value KubernetesReader::FindTopLevelOwner(
 #endif
   const json::Object* metadata = obj->Get<json::Object>("metadata");
 #ifdef VERBOSE
-  LOG(DEBUG) << "FindTopLevelOwner: metadata is " << *metadata;
+  LOG(DEBUG) << "FindTopLevelController: metadata is " << *metadata;
 #endif
   if (!metadata->Has("ownerReferences")) {
 #ifdef VERBOSE
-    LOG(DEBUG) << "FindTopLevelOwner: no owner references in " << *metadata;
+    LOG(DEBUG) << "FindTopLevelController: no owner references in " << *metadata;
```

Comment:
Let's fit in 80 characters (line break before the `<<`).

Reply:
done